lms/static/js/Markdown.Converter.js

```diff
@@ -1285,27 +1285,12 @@ else
         //  attacklab: Utility functions
         //
 
-        var _problemUrlChars = /(?:["'*()[\]:]|~D)/g;

```

Comment:
Looks like our comments crossed... as I noted below, I'm OK with the targeted change you are making, but would like comments in the code explaining where you got the newer method from.